go/porcelain/deploy_test.go

```diff
@@ -130,3 +130,14 @@ func TestConcurrentFileUpload(t *testing.T) {
 		}()
 	}
 }
+
+func TestReadZipRuntime(t *testing.T) {
+	runtime, err := readZipRuntime("../internal/data/hello-world.zip")
```

Comment:
could this file be called something that indicates that this is a sample for a rust function?